code/presets/react-webpack/src/framework-preset-react-docs.ts

```diff
@@ -43,6 +42,10 @@ export const webpackFinal: StorybookConfig['webpackFinal'] = async (config, opti
     return config;
   }
 
+  const { default: ReactDocgenTypescriptPlugin } = await import(
+    '@storybook/react-docgen-typescript-plugin'
+  );
```

Comment:
I took a look at what's generated in `dist` and TSup is leaving the import in place, making it a imperative import trying to load CommonJS. Which I don't think is really valid?

Maybe we should do an `require()` instead?